rktlet/runtime/helpers.go

```diff
@@ -121,10 +121,10 @@ func toContainerStatus(uuid string, app *rkt.App) (*runtimeApi.ContainerStatus,
 	status.FinishedAt = app.FinishedAt
 	status.ExitCode = app.ExitCode
 	status.ImageRef = &app.ImageID
-	status.Image = &runtimeApi.ImageSpec{Image: getImageName(app.CRIAnnotations)}
+	status.Image = &runtimeApi.ImageSpec{Image: getImageName(app.UserAnnotations)}
```

Comment:
@pskrzyns It's because `rkt app add` doesn't require image name, it does `rkt app add $uuid $imageid`, so the result pod manifest could end up with only image ID, and no image name.

Thus we need to store the image name somewhere in the rkt container so that it can be returned later.

I once suggested that we could potentially remove the `image name` field in the CRI in `ContainerStatus` because kubelet should have enough information to know what's the image of the container.